### ISSUE TITLE
CollectionReport - apply css styling to `td` elements

### ISSUE THREAD

**Issue by solo-1234** [alexed1/LightningFlowComponents#704]
**Is your feature request related to a problem? Please describe.**
I'd like to apply padding inside the cells of the table generated by CollectionReport

**Describe the solution you'd like**
An input var to apply css to `td` elements just like we have for `tr`

**Describe alternatives you've considered**
I tried applying padding to the `tr` elements but it didn't do anything

**Additional context**
Screenshot: https://imgur.com/37AmisH

I configured a border using css on `tr`:
```
border: solid;
border-width: 1px;
border-color: lightgray;
```

Thanks!

**Comment by alexed1** [comment]
yep, someone should be able to simply copy the approach we took for tr and th.
